docs/book/starter-guide/pipelines/fetching-pipelines.md

```diff
@@ -141,30 +141,55 @@ zenml pipeline runs list -p <MY_PIPELINE_NAME_OR_ID>
 Each run has a collection of useful metadata which you can access to ensure all runs 
 are reproducible:
 
-#### git_sha
+#### Git SHA
 The Git commit SHA that the pipeline run was performed on. This will only be set 
 if the pipeline code is in a git repository and there are no uncommitted files 
 when running the pipeline.
 ```python
 commit = run.git_sha
 ```
 
-#### status
+#### Status
 The status of a pipeline run can also be found here. There are four 
 possible states: failed, completed, running, cached:
 ```python
 status = run.status
 ```
 
-#### pipeline_configuration
+#### Configuration
 
 The `pipeline_configuration` is a super object that contains all configuration of 
```

Comment:
What is a 'super object'?

Reply:
An object that is really really nice and great 👍

Reply:
HAHA i would rephrase this as `is an object`